ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6072,6 +6080,16 @@ HTWHLSTC:
   - supply_air_temperature_sensor
   - supply_air_temperature_setpoint
 
+HTWHLSS:
```

Comment:
typically, heat wheels control to something (temperature, enthalpy, etc.) this is not a complete enough type. please fix.